code/dynamic_programming/src/number_of_way_to_reach_to_destination_in_maze/countWays.cpp

```diff
@@ -0,0 +1,56 @@
+#include<bits/stdc++.h>
```

Comment:
#1915. Remove using statement as well. Reformat file with clang format and 4 spaces.